src/wasm-linker.h

```diff
@@ -241,7 +244,7 @@ class Linker {
 
   // Support for emscripten integration: generates dyncall thunks, emits
   // metadata for asmConsts, staticBump and initializer functions.
-  void emscriptenGlue(std::ostream& o);
+  void emscriptenGlue(std::ostream& o, bool allowMemoryGrowth);
```

Comment:
better to avoid adding `bool` arguments. could we just call `generateMemoryGrowthFunction` ourselves outside (if this isn't going to called from anywhere else anyhow)? or we could add a parallel method on this class, `emscriptenMemoryGrowthGlue` or `emscriptenGlueMemoryGrowth` (glue as a verb in the latter ;) that we call from outside?